modules/pairIdSystem.js

```diff
@@ -67,14 +66,20 @@ export const pairIdSubmodule = {
       }
     }
 
-    const configParams = (config && config.params) || {};
-    if (configParams && configParams.liveramp) {
-      let LRStorageLocation = configParams.liveramp.storageKey || DEFAULT_LIVERAMP_PAIR_ID_KEY;
-      const liverampValue = pairIdFromLocalStorage(LRStorageLocation) || pairIdFromCookie(LRStorageLocation);
+    const configParams = (config && config.params) ? config.params : {};
+    const cleanRooms = Object.keys(configParams);
 
-      if (liverampValue) {
+    for (let i = 0; i < cleanRooms.length; i++) {
+      const cleanRoom = cleanRooms[i];
+      const cleanRoomParams = configParams[cleanRoom];
+
+      const cleanRoomStorageLocation = cleanRoomParams.storageKey || DEFAULT_STORAGE_PAID_ID_KEYS[cleanRoom];
+      const cleanRoomValue = pairIdFromLocalStorage(cleanRoomStorageLocation) || pairIdFromCookie(cleanRoomStorageLocation);
```

Comment:
```suggestion
    Object.entries(configParams).forEach( ([cleanRoom, cleanRoomParams]) => { 

      const cleanRoomStorageLocation = cleanRoomParams.storageKey || DEFAULT_STORAGE_PAID_ID_KEYS[cleanRoom];
      const cleanRoomValue = pairIdFromLocalStorage(cleanRoomStorageLocation) || pairIdFromCookie(cleanRoomStorageLocation);
```

JS made this a little more convienent :smile: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Suggestion won't fully work as there's probably a closing brace missing